src/thunderbots/software/ai/hl/stp/tactic/move_tactic.h

```diff
@@ -6,34 +6,44 @@
 class MoveTactic : public Tactic
 {
    public:
-    explicit MoveTactic(const Robot& robot);
+    /**
+     * Creates a new MoveTactic
+     *
+     * The MoveTactic will move the assigned robot to the given destination and arrive
+     * with the specified final orientation and speed
```

Comment:
nit; but I think this description of what the `MoveTactic` does should be in the class Javadoc comment, rather then the constructor

Reply:
comment moved